python/cugraph/cugraph/components/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+set(cython_sources connectivity_wrapper.pyx)
```

Comment:
In general I believe we can simplify the CMakeLists.txt in cugraph that use `rapids_cython_create_modules` to the following pattern:

```cmake
rapids_cython_create_modules(
  CXX
  SOURCE_FILES connectivity_wrapper.pyx
  LINKED_LIBRARIES cugraph::cugraph MODULE_PREFIX components_
)
foreach(cython_module IN LISTS RAPIDS_CYTHON_CREATED_TARGETS)
  set_target_properties(${cython_module} PROPERTIES INSTALL_RPATH "\$ORIGIN;\$ORIGIN/../library")
endforeach()
```

Which allows us to remove the `cython_sources` and `linked_libraries` variable. 

As this is a style change, I would priotize it last and would be okay with this style change happening in 21.10 if needed.

Reply:
Got it

Reply:
We could make the same change in other libs too; I think the variable-based approach was just what I used on the first PR where I enabled scikit-build on RMM and it's propagated everywhere. Do you have a preference on how this should look based on how long the list of files/linked libraries is?